python/cucim/src/cucim/skimage/_vendored/_ndimage_morphology.py

```diff
@@ -480,6 +481,7 @@ def binary_dilation(
 
     .. seealso:: :func:`scipy.ndimage.binary_dilation`
     """
+    axes = _util._check_axes(axes, input.ndim)
     structure, structure_shape, symmetric = _prep_structure(
         structure, input.ndim
```

Comment:
I wonder why this statement doesn't use `len(axes)` while others are using it.
```suggestion
        structure, len(axes)
```